domain/src/Item.js

```diff
@@ -21,9 +21,11 @@ function calcAps(item) {
 
 function calcDps(item) {
     let res = (((item.minDmg + item.maxDmg) / 2) * item.aps);
-    return (item.critChance && item.critDmg)
-        ? (res + (item.critChance / 100) * (item.critDmg / 100))
-        : res;
+    if (item.critChance && item.critDmg) {
+        if (item.critChance > 100) item.critChance = 100;
```

Comment:
for future reference - if you are not sure how to implement a certain feature, you must ask the client (me in this case) or at least put a comment in the code with your thoughts. Like so:
```js
        // HACK: i'm not sure if it is possible to reach more than 100% crit 
        // chance in diablo and how it handles it so I just hard limit it here.
        if (item.critChance > 100) item.critChance = 100;
 
```